opm/autodiff/RateConverter.hpp

```diff
@@ -21,13 +21,12 @@
 #ifndef OPM_RATECONVERTER_HPP_HEADER_INCLUDED
 #define OPM_RATECONVERTER_HPP_HEADER_INCLUDED
 
-#include <opm/autodiff/BlackoilPropsAdFromDeck.hpp>
-
 #include <opm/core/props/BlackoilPhases.hpp>
 #include <opm/core/simulator/BlackoilState.hpp>
 #include <opm/core/utility/RegionMapping.hpp>
 #include <opm/core/linalg/ParallelIstlInformation.hpp>
 
+#include <dune/grid/common/gridenums.hh>
```

Comment:
Why is this needed?

Reply:
without it, `RateConverter.hpp` is not autonomous enough to make `test_rateconverter.cpp` compile: it will produce an error for https://github.com/OPM/opm-simulators/blob/master/opm/autodiff/RateConverter.hpp#L500.

Reply:
(before that file was indirectly included via BlackoilPropsAdFromDeck.hpp)